component-interaction/src/app/user-details/user-details.component.ts

```diff
@@ -15,8 +15,14 @@ export class UserDetailsComponent {
 
 	@Output()
 	public onDeleteUser = new EventEmitter<number>()

```

Comment:
```suggestion
	public onDelete = new EventEmitter<number>()

```
onDelete I think is enough, because we are in the user-details component

Reply:
all the actions are on the user